components/raft_log_engine/src/engine.rs

```diff
@@ -173,25 +173,17 @@ impl RaftEngine for RaftLogEngine {
     }
 
     fn has_builtin_entry_cache(&self) -> bool {
-        true
+        false
     }
 
-    fn gc_entry_cache(&self, raft_group_id: u64, to: u64) {
-        self.0.compact_cache_to(raft_group_id, to)
-    }
+    fn gc_entry_cache(&self, _raft_group_id: u64, _to: u64) {}
+
     /// Flush current cache stats.
     fn flush_stats(&self) -> Option<CacheStats> {
-        let stat = self.0.flush_cache_stats();
-        Some(engine_traits::CacheStats {
-            hit: stat.hit,
-            miss: stat.miss,
-            cache_size: stat.cache_size,
-        })
+        None
     }
 
-    fn stop(&self) {
-        self.0.stop();
-    }
+    fn stop(&self) {}
```

Comment:
Shall we support stop RaftLogEngine in future?

Reply:
Raft engine itself doesn't manage any threads anymore, so stop is simply exiting the client process.